meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend

```diff
@@ -19,25 +21,8 @@ FILES:${PN} += " \
 # It is used a lot in the do_install, so variable will be handy
 CFG_FILE="${D}${sysconfdir}/xen/domu.cfg"
 
-CMD_LINE_ROOT_DEVICE = "xvda1"
-CMD_LINE_ROOT_DEVICE:enable_virtio = "vda"
-
-CMD_LINE_PVR = "pvrsrvkm.DriverMode=1"
-CMD_LINE_PVR:enable_virtio = ""
-
-CMD_LINE_EXTRA = "\"root=/dev/${CMD_LINE_ROOT_DEVICE} rw rootwait console=hvc0 cma=256M ${CMD_LINE_PVR}\""
```

Comment:
This looks like a step backwards. Why are you removing this?

Now if anyone want to change something in the command line, they need to do this in all the different .cfg files